modules/mbedtls/Kconfig.tls-generic

```diff
@@ -241,30 +241,41 @@ config MBEDTLS_CIPHER_ALL_ENABLED
 	select MBEDTLS_CIPHER_MODE_CTR_ENABLED
 	select MBEDTLS_CHACHAPOLY_AEAD_ENABLED
 
+config MBEDTLS_CIPHER_SOME_ENABLED
+	bool
+	default y
+	depends on \
+		MBEDTLS_CIPHER_AES_ENABLED || \
+		MBEDTLS_CIPHER_CAMELLIA_ENABLED || \
+		MBEDTLS_CIPHER_DES_ENABLED || \
+		MBEDTLS_CIPHER_ARC4_ENABLED || \
+		MBEDTLS_CIPHER_CHACHA20_ENABLED || \
+		MBEDTLS_CIPHER_BLOWFISH_ENABLED
```

Comment:
It's preexisting, but as far as I know neither ARC4 (`MBEDTLS_ARC4_C`) nor BlowFish (`MBEDTLS_BLOWFISH_C`) are supported in MbedTLS.

Reply:
And, just for reference, I just realized that ARIA (MBEDTLS_ARIA_C) is instead supported, but not listed at all in Kconfig.tls-generic/config-tls-generic.h.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/73222